modules/swagger-codegen/src/main/resources/nodejs-google-cloud-functions/index.mustache

```diff
@@ -0,0 +1,44 @@
+'use strict';
```

Comment:
Sure, I don't care the exact file location as long as the code works. But thank you for telling me!